features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Listing/Listing.js

```diff
@@ -26,18 +26,13 @@ import Loading from '../../Base/Loading/Loading'
 import ResourceNotFound from "../../Base/Errors/ResourceNotFound";
 import Grid from 'material-ui/Grid';
 import Typography from 'material-ui/Typography';
-import BottomNavigation, {BottomNavigationAction} from 'material-ui/BottomNavigation';
-import ListIcon from 'material-ui-icons/List';
-import GridOnIcon from 'material-ui-icons/GridOn';
 
-import Table, {TableBody, TableCell, TableHead, TableRow} from 'material-ui/Table';
-import Paper from 'material-ui/Paper';
+import Table, { TableBody, TableCell, TableHead, TableRow, TableSortLabel } from 'material-ui/Table';
```

Comment:
recommended to use as 

import TableBody from '@material-ui/core/TableBody';
import TableCell from '@material-ui/core/TableCell';
import TableHead from '@material-ui/core/TableHead';
import TableRow from '@material-ui/core/TableRow';

Reply:
Already fixed in master branch with PR : https://github.com/dushaniw/carbon-apimgt/commit/70fd07f000ed08fe6781e1e38744c41840722c0d#diff-7dd21651eb3f7d07cd69cdb0f4d4f3b9R30

Reply:
@dakshika , Noted and agreed, We have noticed that these import statments are not tree shaking friendly , causing the include whole 3rd party library into the final build. Thanks for pointing out.